packages/mds-cache/index.ts

```diff
@@ -168,7 +168,7 @@ async function hreads(suffixes: string[], device_ids: UUID[]): Promise<CachedIte
         resolve(
           replies.map((flat, index) => {
             if (flat) {
-              const flattened = { ...flat, device_id: device_ids[index % device_ids.length] }
+              const flattened = { ...flat, [`${prefix}_id`]: ids[index % ids.length] }
```

Comment:
Would it have made more sense to have a generic property name rather than constructing one?

Reply:
I'm not sure what you mean by that. Could you elaborate please?

Reply:
The property is going to be either `device_id` or `provider_id` ... just asking if it should be the same name always.

Reply:
If I'm understanding your suggestion correctly, you're proposing consolidating `device_id` and `provider_id` into some constant `element_id` field?

I'm not sure about the usage of this key in other places, so my inclination would be to leave it as is for now. @avatarneil perhaps you could shed some light onto why we'd want two different key names vs. consolidating into one?

Reply:
Alright, we tried putting `entity_id` in the object so it would be the same for either branch, but that broke some seemingly unrelated tests. I'd like to push to get this up so we can test the performance, then we can revisit this key naming issue in a subsequent change set. How does that sound @drtyh2o?